lonestar/analytics/CMakeLists.txt

```diff
@@ -1,9 +1,29 @@
+function(app_analy_cpu name bin_name)
```

Comment:
It seems like a lot of extra complexity to introduce these types of functions for the sole purpose of prefixing targets when the directory structure in the build tree is something only relevant to developers (and subject to future change as we reorganize things) while most users would be using the flattened structure in the installed bindir.

Reply:
Do we expect users to use installed versions of the apps?

Reply:
I don't think so. The users are mostly interested in using Galois as a library if they have their own applications to be Galoised.

Reply:
Installing the apps seems very unlikely to me; most people probably don't need a graph analytics routine they can call on the fly.

Reply:
Insofar as most users of this repo are other developers, I agree that using the build tree is a convenient way to refer to executables, but, if I were instead someone interested in installing a suite of graph analytics applications for myself and others, I would naturally assume that the installed executables are the primary artifact. (In the same way as I would think that the installed library and header files are the primary artifact of this repo for developers.)

Reply:
Agree